config/locales/in_person_proofing/en.yml

```diff
@@ -19,9 +19,10 @@ en:
         return_to_partner_link: sign out and return to %{sp_name}
         what_to_expect: What to expect at the Post Office
       cta:
-        button: Try in person
-        prompt_detail: You may be able to verify your ID in person at a participating
-          Post Office near you.
+        button: Try at a Post Office
+        prompt_detail: You’ll start this process by entering your information online,
+          and continue to verify your identity in person at a participating Post
+          Office near you.
       expect:
```

Comment:
No other instances of `in_person_proofing.body.cta.button` found except in spec files

<img width="291" alt="Screenshot 2023-10-31 at 3 31 43 PM" src="https://github.com/18F/identity-idp/assets/125507397/a1173903-e0f2-4787-a4a7-d3ac9c5348fc">


No other instances of `in_person_proofing.body.cta.prompt_detail` found except in spec a file

<img width="367" alt="Screenshot 2023-10-31 at 3 32 20 PM" src="https://github.com/18F/identity-idp/assets/125507397/efa965ee-31e6-4d81-a1fa-12d13cdc2167">